app/models/provisioning_template.rb

```diff
@@ -140,7 +140,7 @@ def self.build_pxe_default(renderer)
           Foreman::Logging.exception("Cannot render '#{global_template_name}'", exception)
           error_msgs << "#{exception.message} (#{kind})"
         end
-        return [422, error_msg] unless error_msg.empty?
+        return [:unprocessable_entity, error_msgs.join(', ')] unless error_msgs.empty?
```

Comment:
@lzap - Correct me if I understood incorrectly, but looks like error_msg was always empty here because it is only set in the other branch of the if statement. tbh i'd rather put the return in the rescue block that needs to return rather then after it, kind of unexpected in the middle of this else block.